ts/features/idpay/screens/PdndAcceptance.tsx

```diff
@@ -0,0 +1,100 @@
+import { List, Text, View } from "native-base";
```

Comment:
Move to "ts/features/idpay/**onboarding**/screens" and rename it to PDNDPrerequisites.tsx